README.md

```diff
@@ -38,27 +40,51 @@ In order to expose the hypervisor UI, generate a config file with `--is-hypervis
 $ skywire-cli visor gen-config --is-hypervisor
 ```
 
+Docker container will create config automatically for you, should you want to run it manually, you can do:
+
+```bash
+$ docker run --rm -v <YOUR_CONFIG_DIR>:/opt/config \
```

Comment:
We should mount the config to `/opt/skywire/` as I think that is the actual path on our other installations.